src/System.Runtime.InteropServices.RuntimeInformation/src/project.json

```diff
@@ -6,10 +6,12 @@
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
-    "System.Threading": "4.0.0"
+    "System.Threading": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Reflection.Extensions": "4.0.0"
```

Comment:
This is very inefficient because of it will create instances of all custom attributes, and then filters them out using Linq.

Is it possible to use `ICustomAttributeProvider.GetCustomAttributes(Type attributeType, bool inherit)` here instead?